specs/credentialmanagement/index.src.html

```diff
@@ -238,7 +238,7 @@ <h4 id="examples-password-signin">Password-based Sign-in</h4>
     <div class="example">
       <pre>
         navigator.<a attribute>credentials</a>.<a idl lt="get()">get</a>({
-          "<a idl>types</a>": [ "password" ]
+          "<a idl>filters</a>": { "password" : {} }
```

Comment:
It's important insofar as it's the use case I actively care about, and the use case I expect to see (in the short term) as 90% of the API's implementation in web developer's code.

If we keep the distinction between `types` and `filters`, we can deal with filterless types more cleanly. That is, the password case would remain `.get({ types: [ 'password' ] })`, as no filter would be required..